pwem/emlib/image/__init__.py

```diff
@@ -24,17 +24,5 @@
 # *
 # **************************************************************************
 
-import traceback
-
-# FIXME: This is a bypass until xmipp is replaced to xmippLib
-# FIXME:   in all imports related to the C++ binding
-
-print("\n >>> WARNING: 'import xmipp' is deprecated for the xmipp binding. "
-      "Please, use 'import xmippLib'")
-stackList = traceback.extract_stack()
-for stackLine in stackList:
-    if 'import xmipp' in stackLine[3] or 'from xmipp import' in stackLine[3]:
-        print("  > Please change 'import xmipp' to 'import xmippLib' in\n"
-              "    File: '%s', line %d.\n" % (stackLine[0], stackLine[1]))
-
-from xmippLib import *
```

Comment:
This file was temporary since some time ago the binding was through `import xmipp` and we want to warn developers that the correct way is `import xmippLib`, if they still using that former way.

To be honest, I've never seen that waning triggered, so I will say we can remove this file. If you don't think so, I would keep the warning to be able to remove it in the future.

Reply:
Hi @dmaluenda, the module xmip-ghost is gone! I think git diff was confused here because I copy/paste the header file. I agree with you that it has been quite a while with that warning, so if it appears, we should address in the specific case.

Reply:
It is gone now! Thanks for the feeback.

Reply:
Sorry you right. Well, the ghost is gone but the _libNone replace it, this is not the point. However, since the file name changed from `.../xmipp.py` to `.../image/__init__.py`, what is gone is the option to call the binding by `import xmipp`. This is my first sugestion so, it's done.

Sorry, I haven't seen the file-name changing in the diff and I simply focused on the content.

Great!